lib/identity/client.js

```diff
@@ -166,6 +166,27 @@ class Client extends PartialClient {
     return resultQuery
   }
   /**
+   * getLatestSecrets
+   *
+   * @param {string} secretName   The name of the Secret given by User.
+   * @param {string} secretType   The type of the Secret chosen by User.
+   *
+   *
+   * @return {Promise<Record>}
+   */
+  async getLatestSecret(secretName, secretType) {
+    const request = new Search(true)
+    let searchType = `tozny.secret.${SECRET_UUID}.${secretType}.${secretName}`
+    request.match({ type: searchType }, 'AND', 'EXACT')
+    let resultQuery = await this.storage.search(request)
+    let resultList = await resultQuery.next()
+    console.log(resultList)
+    if (resultList.length > 0) {
+      resultList[resultList.length - 1]
```

Comment:
I think we want to be able to grab item 0 here such that we know even if there are multiple "pages" of results we have the newest